test-project/Assets/EditmodeTests/Subscriptions/SubscriptionsTestBase.cs

```diff
@@ -0,0 +1,50 @@
+using Improbable.Gdk.Core;
+using Improbable.Gdk.Subscriptions;
+using NUnit.Framework;
+using UnityEngine;
+
+namespace Improbable.Gdk.EditmodeTests.Subscriptions
+{
+    public class SubscriptionsTestBase
+    {
+        protected WorkerInWorld WorkerInWorld;
+        protected SpatialOSReceiveSystem ReceiveSystem;
+        protected RequireLifecycleSystem RequireLifecycleSystem;
+        protected MockConnectionHandler ConnectionHandler;
+        protected EntityGameObjectLinker Linker;
+
+        [SetUp]
+        public virtual void Setup()
+        {
+            var connectionBuilder = new MockConnectionHandlerBuilder();
+            ConnectionHandler = connectionBuilder.ConnectionHandler;
+
+            WorkerInWorld = WorkerInWorld
+                .CreateWorkerInWorldAsync(connectionBuilder, "TestWorkerType", new LoggingDispatcher(), Vector3.zero)
+                .Result;
+
+            var world = WorkerInWorld.World;
+            ReceiveSystem = world.GetExistingSystem<SpatialOSReceiveSystem>();
+            RequireLifecycleSystem = world.GetExistingSystem<RequireLifecycleSystem>();
+            Linker = new EntityGameObjectLinker(world);
+        }
+
+        [TearDown]
+        public virtual void TearDown()
+        {
+            WorkerInWorld.Dispose();
+        }
+
+        protected GameObject CreateAndLinkGameObjectWithComponent<T>(long entityId) where T : MonoBehaviour
+        {
+            var gameObject = new GameObject("TestGameObject");
+            gameObject.AddComponent<T>();
```

Comment:
`AddComponent` returns the Component, so enabled can be set to false there.

Reply:
Ye